bundles/org.openhab.binding.wemo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -455,6 +507,10 @@
 		<label>Auto OFF Time</label>
 		<description>Time when a WeMo enabled Heater should switch off</description>
 		<category>Time</category>
+		<tags>
+			<tag>Status</tag>
```

Comment:
Channel is writable.
```suggestion
			<tag>Control</tag>
```

?

Reply:
Maybe even a Setpoint ?? but I won't quibble..